BinaryEncoding.md

```diff
@@ -81,7 +81,7 @@ Yes:
   - A table (sorted by offset) containing, for each type which has opcodes:
     + A standardized string literal [type name](AstSemantics.md#local-and-memory-types).
       The index of a type name in this table is referred to as a type ID
-    + 32-bit offset of its opcode table within the section
+    + 64-bit offset of its opcode table within the section
```

Comment:
@waywardmonkeys it's still not clear that LEB128 or any other variable-length integer encoding pays off after applying generic encoding such as gzip, lzham or brotli. We're optimizing a multi-variable encoding approach, and I'm pretty wary of early optimization for now!